react/week4/my-app/src/FindUser.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import context from "./context";
+export default function FindUser() {
+  const { handelUserSearch } = React.useContext(context);
+  const form = (
+    <form>
```

Comment:
You don't need the form here, it's really only necessary if you're passing an `onSubmit` to handle submission of your inputs.